internal/compiler/program.go

```diff
@@ -862,7 +862,7 @@ func (p *Program) verifyProjectReferences() {
 		}
 		refOptions := config.CompilerOptions()
 		if !refOptions.Composite.IsTrue() || refOptions.NoEmit.IsTrue() {
-			if len(config.FileNames()) > 0 {
+			if len(parent.FileNames()) > 0 {
```

Comment:
The old code said:

```
const inputs = parent ? parent.commandLine.fileNames : rootNames;
if (inputs.length) {
```

I assume `parent` simply always exists?

Reply:
Seems to be the case in Corsa: this function starts with `ref := parent.ProjectReferences()[index]`, which would crash if `parent` is `nil`.

Reply:
thanks. yes with being able to use and store parsedCommandLine in program, we always have parent 
I had this in #1484  fixed incorrectly too (was always checking program's file names) so thank you for doing the right fix.